assets/sass/_theme/_utils.sass

```diff
@@ -8,18 +8,17 @@
     $remSize: $size / 16 / 2
     @return #{$remSize}rem
 
-@mixin in-page-with-aside
-    body:not(.full-width) aside ~ .blocks &
+@mixin in-page-with-nav
```

Comment:
Pas clair, il y a une nav en haut de toutes les pages. 
in-page-with-toc ?
ou bien laisser in-page-with-aside ?